db/migrate/20220823180758_remove_posts_counter_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemovePostsCounterFromUsers < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :users, :postsCounter, :integer
+  end
+end
```

Comment:
## [OPTIONAL]

- [ ] Kindly solve these errors in your migration. You are removing columns that do not exist. 

![image](https://user-images.githubusercontent.com/65068771/186272983-461aa40b-011d-481f-9539-7dc32a02d8ee.png)

Reply:
Yes, the reason is I delete the files after I remove columns.